etna/pipeline/pipeline.py

```diff
@@ -54,11 +60,18 @@ def _forecast(self) -> TSDataset:
         if self.ts is None:
             raise ValueError("Something went wrong, ts is None!")
 
-        if isinstance(self.model, DeepBaseModel):
-            future = self.ts.make_future(future_steps=self.model.decoder_length, tail_steps=self.model.encoder_length)
-            predictions = self.model.forecast(ts=future, horizon=self.horizon)
+        if isinstance(self.model, get_args(ContextRequiredModelType)):
```

Comment:
Why do wee need get_args here?

Reply:
Because we can't make `isinstance(self.model, ContextRequiredModelType)` because `ContextRequiredModelType` isn't a class it is a `Union`.